src/Templates/src/templates/maui-blazor/Components/Layout/NavMenu.razor

```diff
@@ -1,6 +1,6 @@
 ﻿<div class="top-row ps-3 navbar navbar-dark">
     <div class="container-fluid">
-        <a class="navbar-brand" href="">BlazorWeb-CSharp</a>
```

Comment:
This was a copy/paste accident when updating the templates to the latest Blazor. This should have used the MAUI project name substitution so that it gets replaced with whatever the user called their project.